modules/labs-react/search-form/lib/SearchForm.tsx

```diff
@@ -118,173 +117,318 @@ function getInputColors(theme: SearchThemeAttributes, isFocused?: boolean) {
   };
 }
 
-const formCollapsedBackground = colors.frenchVanilla100;
+const formCollapsedBackground = '#fff';
 
-const maxWidth = 480;
-const minWidth = 120;
-
-const StyledSearchForm = styled('form')<
-  Pick<SearchFormProps, 'isCollapsed' | 'rightAlign' | 'grow'> & Pick<SearchFormState, 'showForm'>
->(
-  {
+const searchFormStencil = createStencil({
+  vars: {
+    minWidth: px2rem(120),
+    maxWidth: px2rem(480),
```

Comment:
Setting the defaults here, i don't think Search warrants cascading but open to thoughts.

Reply:
I think cascade barriers make more sense when a component is able to wrap another component of the same type. For example, a `Card` can wrap another `Card` and we don't want the variables set for the outer `Card` to change the inner `Card`.

A cascade barrier prevents the following from working:

```jsx
<div cs={{[searchFormStencil.vars.minWidth]: '300px'}}>
  <SearchForm ...>
```

There isn't an apparent reason why setting CSS variables for other things works, but not for `minWidth`. At first, I thought the default would be defining defaults for variables, but now I think the default should be fallbacks.

Reply:
ah makes sense, i can update to use cssVar instead, good call